include/aml.h

```diff
@@ -129,45 +129,44 @@ struct aml_area_ops {
 	/**
 	 * Building block for coarse grain allocator of virtual memory.
 	 *
-	 * @param[in] data: Opaque handle to implementation specific data.
-	 * @param[in] size: The minimum size of allocation.
+	 * @param[in] data: opaque handle to implementation specific data.
+	 * @param[in] size: the minimum size of allocation.
 	 *        Is greater than 0. Must not fail unless not enough
 	 *        memory is available, or ptr argument does not point to a
 	 *        suitable address.
 	 *        In case of failure, aml_errno must be set to an appropriate
 	 *        value.
-	 * @param opts: Opaque handle to pass additional options to area
+	 * @param[in] opts: opaque handle to pass additional options to area
 	 *        mmap hook. Can be NULL and must work with NULL opts.
 	 * @return a pointer to allocated memory object.
 	 **/
-	void* (*mmap)(const struct aml_area_data  *data,
-		      size_t                       size,
-		      struct aml_area_mmap_options *opts);
+	void *(*mmap)(const struct aml_area_data *data,
+	              size_t size,
+	              struct aml_area_mmap_options *opts);
 
 	/**
 	 * Building block for unmapping of virtual memory mapped with mmap()
 	 * of the same area.
 	 *
-	 * @param data: An opaque handle to implementation specific data.
-	 * @param ptr: Pointer to data mapped in physical memory. Cannot be
+	 * @param[in] data: an opaque handle to implementation specific data.
+	 * @param[in] ptr: pointer to data mapped in physical memory. Cannot be
 	 *        NULL.
-	 * @param size: The size of data. Cannot be 0.
+	 * @param[in] size: the size of data. Cannot be 0.
 	 * @return: AML_AREA_* error code.
 	 * @see mmap()
 	 **/
-	int (*munmap)(const struct aml_area_data *data,
-		      void                       *ptr,
-		      size_t                      size);
+	int (*munmap)(const struct aml_area_data *data, void *ptr, size_t size);
 
 	/**
 	 * Print the implementation-specific information available
-	 * @param stream the stream to print to
-	 * @param prefix a prefix string to use on all lines
-	 * @param data non-NULL handle to area internal data.
+	 * @param[out] stream: the stream to print to
```

Comment:
```suggestion
	 * @param[in] stream: the stream to print to
```
That's actually an `in`, even though we "write" into it